### PR TITLE
Syntax errors, fixed root identification

### PR DESCRIPTION
**Pull request by davidarps**
This pull request has two components. 
- First, the codebase has some minor syntax and import errors, without which the code does not run. 
- Second, the ACL paper describes that dependency tree roots are identified by finding the word at which p(root) is highest across the sentence. However, the current code identifies the root by taking the word at which the logit for root is highest. Leads to slightly different results on some masked models, In some autoregressive models, logits at the sentence start are really high, and then the first words are selected as the root.

### CODE REVIEW

**Comment by personads** [comment]
Hi David,

Thanks so much for putting in the time and effort! I will merge your commits into main.

Best,
Max
